src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -14,26 +14,35 @@
 
 namespace Microsoft.PowerShell.EditorServices.Services.PowerShellContext
 {
+    using System.IO;
     using System.Management.Automation;
 
     internal class PSReadLinePromptContext : IPromptContext
     {
-        private const string ReadLineInitScript = @"
+        private static readonly string _psReadLineModulePath = Path.Combine(
+            Path.GetDirectoryName(typeof(PSReadLinePromptContext).Assembly.Location),
+            "..",
+            "..",
+            "..",
+            "PSReadLine");
+
+        private static readonly string ReadLineInitScript = $@"
             [System.Diagnostics.DebuggerHidden()]
             [System.Diagnostics.DebuggerStepThrough()]
             param()
-            end {
+            end {{
                 $module = Get-Module -ListAvailable PSReadLine |
-                    Where-Object { $_.Version -gt '2.0.0' -or ($_.Version -eq '2.0.0' -and -not $_.PrivateData.PSData.Prerelease) } |
+                    Where-Object {{ $_.Version -ge '2.0.2' }} |
```

Comment:
Might be worth taking the version here out into a constant

Reply:
I'll do that next time 😅 I really don't want to test on my SAW again lol